src/main/scala/dpla/ingestion3/mappers/providers/DcMapping.scala

```diff
@@ -0,0 +1,138 @@
+package dpla.ingestion3.mappers.providers
+
+import java.net.URI
+
+import dpla.ingestion3.enrichments.normalizations.StringNormalizationUtils._
+import dpla.ingestion3.enrichments.normalizations.filters.{DigitalSurrogateBlockList, FormatTypeValuesBlockList}
+import dpla.ingestion3.mappers.utils.{Document, IdMinter, Mapping, XmlExtractor}
+import dpla.ingestion3.model.DplaMapData.{ExactlyOne, ZeroToOne}
+import dpla.ingestion3.model._
+import dpla.ingestion3.utils.Utils
+import org.json4s.JValue
+import org.json4s.JsonDSL._
+
+import scala.xml._
+
+
+class DcMapping extends Mapping[NodeSeq] with XmlExtractor with IdMinter[NodeSeq] {
+  // ID minting functions
+  override def useProviderName(): Boolean = false // TODO confirm prefix
```

Comment:
Do these TODOs need addressing?

Reply:
I'm not sure if we settled on using prefixes for all providers going forward as the default base DPLA id.  I think this can be addressed after initial CQA review